dark/fasta.py

```diff
@@ -85,11 +85,15 @@ class FastaReads(Reads):
     @param checkAlphabet: An C{int} or C{None}. If C{None}, alphabet checking
         will be done on all reads. If an C{int}, only that many reads will be
         checked. (Pass zero to have no checks done.)
+    @param upperCase: If not C{None}, the reads will be converted to upper
+        case.
     """
-    def __init__(self, _file, readClass=DNARead, checkAlphabet=None):
+    def __init__(self, _file, readClass=DNARead, checkAlphabet=None,
+                 upperCase=None):
```

Comment:
The default value should be `False`, as it's a boolean. It doesn't matter really, but it makes things a little clearer for the reader.